.ci/packaging.groovy

```diff
@@ -133,7 +133,7 @@ pipeline {
             }
           }
         }
-        stage('DRA Staging') {
+        stage('DRA Staging (release branches only)') {
```

Comment:
```suggestion
        stage('DRA Release Staging') {
```

It is annoying that it is truncated, maybe just putting Release in the name will help and it won't be truncated?

We could also go back to the original wording, I don't have a strong preference at this point. Pick whichever of this or your first suggestion you like and merge it :)